docker-compose.yml

```diff
@@ -9,19 +9,29 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: password
       POSTGRES_DB: vault
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      interval: 10s
+      timeout: 5s
+      retries: 10
     volumes:
       - pgdata:/var/lib/postgresql/data
-    ports:
-      - "5432:5432"
+    # ports:
+    #   - "5432:5432"
```

Comment:
why comment out all of the ports?

Reply:
They are not required. For development sure. however, in production all queries must go through your server. As they all exist on the same network access is guaranteed. Port mapping is only to expose it to the bridged network form docker.

Reply:
Basically, left them there for people who may want to connect directly. Not needed otherwise

Reply:
We could remove them and add to the documentation instead. Up to you guys

Reply:
> Basically, left them there for people who may want to connect directly. Not needed otherwise

Hmm, I tend to think that we should just keep them for local dev convenience and let folks who plan to self-host modify it if they so choose.

Reply:
done